packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -113,7 +113,7 @@ const metadata: Meta<TableArgs> = {
                 'https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/screen/d389dc1e-da4f-4a63-957b-f8b3cc9591b4/specs/'
         },
         actions: {
-            handles: []
+            handles: ['action-menu-beforetoggle', 'action-menu-toggle']
         }
     },
```

Comment:
How hard / worthwhile would it be to add a story (possibly in the matrix tests) that renders the table with the menu open?

Reply:
I found a way to do this, but I think I'm going to have to make some changes to the way our fixed theme stories are rendered (right now the viewport height is incorrect, which results in the menu button opening off screen). I'll have to dig into it more, but assuming I'm right, the fix will likely cause a chromatic change in all fixed themes stories. I'll create an issue to track this and resolve it as part of a separate PR.

Reply:
#1058